app/schemas/models/classroom.schema.js

```diff
@@ -102,6 +103,7 @@ _.extend(ClassroomSchema.properties, {
       })
   }, {}),
   stats: c.object({ additionalProperties: true }),
+  initializedOuterStats: { type: 'boolean', default: true, description: 'whether the classroom.stats for the classroom is initialized' },
```

Comment:
we could just initializeStats in post hook of class save or create instead of storing this flag